### DISCUSSION TITLE
Integration with FTB chunks

### DISCUSSION THREAD

**Discussion by pietru2004** [ldtteam/minecolonies#10034] [Ideas]
Could there be integration with FTB chunks? I am thinking of something that would disallow people to claim colony chunks or colony to claim chunk if it has been claimed by other player assuming that player is not owner of colony. Also colony chunks probably wouldn't go into chunk claim limit of ftb chunks

**Comment by BrokenToaster999** [comment]
oh my god yes. Me and my friend are currently facing an issue where a person started a colony next to our claimed border and now half of our claimed chunks are just unuable by anyone.

**Comment by LZeugirdor** [comment]
This, I can't access any of my blocks now that someone made a colony, it's overriding my ftbchunks claim.
